manager/api/transport.go

```diff
@@ -247,10 +248,19 @@ func decodeView(_ context.Context, r *http.Request) (interface{}, error) {
 }
 
 func decodeList(_ context.Context, r *http.Request) (interface{}, error) {
+	offset, err := strconv.Atoi(r.FormValue("offset"))
```

Comment:
Please don't use form values. Instead, use the following approach:

```golang
q := r.URL.Query()
q.Get("offset")
q.Get("limit")
```

Check out the relevant godocs:
- https://golang.org/pkg/net/url/#URL.Query
- https://golang.org/pkg/net/url/#Values.Get